client/sites-dashboard/components/sites-ellipsis-menu.tsx

```diff
@@ -66,44 +84,59 @@ const ManagePluginsItem = ( { site, recordTracks }: SitesMenuItemProps ) => {
 			}
 		>
 			{ label }
-		</PopoverMenuItem>
+		</MenuItemLink>
 	);
 };
 
 const HostingConfigItem = ( { site, recordTracks }: SitesMenuItemProps ) => {
 	const { __ } = useI18n();
 
 	return (
-		<PopoverMenuItem
+		<MenuItemLink
 			href={ getHostingConfigUrl( site.slug ) }
 			onClick={ () => recordTracks( 'calypso_sites_dashboard_site_action_hosting_config_click' ) }
 		>
 			{ __( 'Hosting configuration' ) }
-		</PopoverMenuItem>
+		</MenuItemLink>
 	);
 };
 
-const alignExternalIcon = css`
-	.gridicons-external {
-		top: 0px;
-	}
-`;
+const ExternalGridIcon = styled( Gridicon )( {
+	top: '-1px',
+	marginLeft: '4px',
```

Comment:
Let's go with logical properties:

```suggestion
	insetBlockStart: '-1px',
	marginInlineStart: '4px',
```